templates/extra/faq.html

```diff
@@ -5,16 +5,16 @@
     <div class="break">
         <hr>
     </div>
-    <div class="extra-padding">
+    <div class="extra-padding-wide">
         <ul class="vertical menu" data-accordion-menu>
-            <li>
+            <li class="left-align">
```

Comment:
So I updated this but the original p text align overrides this so I can't get the list to work. 

I'm going to go back and re-do all of the alignment on the site. I think it would make sense to have default be left. The challenge is nearly everything on the site is using that default so it's going to take a pretty good amount of time to re-do everything. 

Are you okay with leaving this for the current PR and I can submit another one with these updates?